### PR TITLE
Simple modular configuration refactoring

### PR DESCRIPTION
**Pull request by Aladoro**
Refactored the hydra configurations to be divided into agent-specific (_agent_cfg_) and environment-specific (_env_cfg_), to ease experimentation. In particular, the only argument that needs to be overridden for any of the experiments with DrQ-v2 is env_cfg.

I have also added the relative configurations for all medium benchmark tasks, following the details from the paper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ conda activate drqv2
 
 Train the agent:
 ```sh
-python train.py task=quadruped_walk
+python train.py env_cfg=quadruped_walk
```
Comment (denisyarats) [inline README.md:75]: let's rename it to just `task`

File: cfgs/env_cfg/acrobot_swingup.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# task settings
+task: acrobot_swingup
+frame_stack: 3
```
Comment (denisyarats) [inline cfgs/env_cfg/acrobot_swingup.yaml:3]: hm there is a lot of overlap going on, only a few arguments are changing. I wonder if there is a way to only have a task specific overrides and still preserve the default config.

**Review by denisyarats — CHANGES_REQUESTED** [review]
Thanks for submitting this!

I think it is way too complicated and convoluted at the moment. I wonder if there is a way to simplify it. Specifically, it would be nice if we still have the default config as before, but only have a folder with task specific overrides. Right now there is a lot of duplication going on and only a few parameters are changing, it would be best if we can preserve this simplicity.

**Review by denisyarats — COMMENTED** [review]
this looks good now, thanks!

I think the humanoid tasks are missing (maybe some others too), could you please add those as well? otherwise looks great.

**Comment by Aladoro** [comment]
Done, added configurations for  all easy and hard benchmark tasks (before I included only the medium benchmark tasks)! Let me know if any other change is required ^^

**Comment by denisyarats** [comment]
thanks, this looks good. will merge now.

**Pull request merged**